code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -496,6 +496,20 @@ This is here to make the tiles around the station mininuke change when it's arme
 	poi_list |= src
 	START_PROCESSING(SSobj, src)
 
+/obj/item/weapon/disk/nuclear/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/claymore/highlander))
```

Comment:
it's pretty snowflakey but honestly i dont see any real better way of doing it except maybe checking for a nuke disk in the sword's attack or something